packages/components/organisms/f-content-cards/src/components/templates/Image.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <div :class="$style['c-content-cardImage--wrapper']">
+        <img
+            data-test-id="content-card-background-image"
+            :class="$style['c-content-cardImage']"
+            :src="backgroundImageUrl"
+            alt="hey">
```

Comment:
Should this be cleaned up?

Reply:
yes good catch will clean up as part of my next PR.